call_service/client.py

```diff
@@ -0,0 +1,29 @@
+import glob
+from os.path import dirname, basename, isfile, join
+from snet_sdk import SnetSDK
+
+from config.config import config
+from stub import *
+
+
+class Client:
+    def __init__(self):
+        self.mapping = {
+            'snet': {'example-service': {'stub': example_service_pb2_grpc.CalculatorStub}}}
```

Comment:
Lets build the mapping in a separate file that can be auto-generated

Reply:
mapping is separated from client code